chrome/content/zotero-better-bibtex/schomd.coffee

```diff
@@ -117,8 +191,10 @@ Zotero.BetterBibTeX.schomd.citation = (citekeys, {style, libraryID} = {}) ->
 
   return '' if itemIDs.length == 0
 
-  url = "http://www.zotero.org/styles/#{style ? 'apa'}"
-  style = Zotero.Styles.get(url)
+  urlstyle = "#{style ? 'apa'}"
+  style = Zotero.Styles.get("http://www.zotero.org/styles/#{urlstyle}")
+  style = Zotero.Styles.get("http://juris-m.github.io/styles/#{urlstyle}") unless style != false
```

Comment:
You tell me and we'll both know. What I do know is that:
    unless style?
... did not work right, because it's not "null" or "undefined", but "false" that is returned to here.